<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="sopLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>sopcast</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sopInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="sopButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="urlLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel List Url</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Static Port</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSpinBox" name="inSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>In Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="outLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Out Port</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="outSpinBox">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButon">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
